src/workflows-app/FeaturedWorkflows.ts

```diff
@@ -16,17 +15,19 @@ import { useCancellation, useOnMount, usePollingEffect } from 'src/libs/react-ut
 import { AppProxyUrlStatus, workflowsAppStore } from 'src/libs/state';
 import { withBusyState } from 'src/libs/utils';
 import { WorkspaceWrapper } from 'src/libs/workspace-utils';
+import { AppDetails } from 'src/pages/workspaces/hooks/useAppPolling';
+import { CloudEnvironmentDetails } from 'src/pages/workspaces/hooks/useCloudEnvironmentPolling';
 import { ImportWorkflowModal } from 'src/workflows-app/components/ImportWorkflowModal';
 import { WorkflowCard, WorkflowMethod } from 'src/workflows-app/components/WorkflowCard';
 import { FeaturedWorkflow, featuredWorkflowsData } from 'src/workflows-app/fixtures/featured-workflows';
-import { doesAppProxyUrlExist, loadAppUrls } from 'src/workflows-app/utils/app-utils';
+import { doesAppProxyUrlExist, loadAppUrls, loadingYourWorkflowsApp } from 'src/workflows-app/utils/app-utils';
 import { CbasPollInterval } from 'src/workflows-app/utils/submission-utils';
 
 type FeaturedWorkflowsProps = {
   name: string;
   namespace: string;
   workspace: WorkspaceWrapper;
-  analysesData: AnalysesData;
+  analysesData: AppDetails & CloudEnvironmentDetails;
```

Comment:
Is there a reason we are not updating the type of `AnalysesData` to reflect this `AppDetails & CloudEnvironmentDetails` value? It seems to me that having this `AppDetails & CloudEnvironmentDetails` everywhere instead of a specific type for analysesData would make it very annoying to update every instance in the future (as we can already see here)

Reply:
It seems to me that this definition https://github.com/DataBiosphere/terra-ui/blob/ad3d9d28d7fe9d6c3e67f4d82b36199df9dfba83/src/pages/workspaces/workspace/WorkspaceContainer.ts#L64 should refer to the existing AnalysesData type and all updates should be made to that singular type. Due to import cycles the type might need to be moved to a different file (or perhaps just defined within the workspace wrapper file itself)